src/components/editPostModal/index.js

```diff
@@ -2,17 +2,28 @@ import { useState } from "react"
 import useModal from "../../hooks/useModal"
 import './style.css'
 import Button from '../button'
+import { patch } from '../../service/apiClient'
 
-const EditPostModal = () => {
+
+// passing current post ID and message content through props
+const EditPostModal = ({ id, content, setTriggerUpdate}) => {
     const { closeModal } = useModal()
     const [message, setMessage] = useState(null)
-    const [text, setText] = useState('')
+    const [text, setText] = useState(content)
 
     const onChange = (e) => {
         setText(e.target.value)
     }
 
     const onSubmit = () => {
+        // sends patch request to update current message with edited message
```

Comment:
Remove comment

Reply:
Comments in this file our team did not make... maybe another team left comments and when we pulled changes it got into our branch somehow and it looks like we left them... In any case, they should be deleted.